docs/smpp.rst

```diff
@@ -84,11 +83,11 @@ TLV fields
 
 We support the following common TLV fields for `submit_sm` as well as one our own custom for use with message classes.
 
-======== ======================= ======== ========================
-Tag      Name                    Size     Description
-======== ======================= ======== ========================
-0x0005   dest_addr_subunit       1 byte   For "flash" sms
-0x0204   user_message_reference  2 bytes  User assigned reference for delivery reports
-0x0424   message_payload         1 byte   For sending messages longer than 255 octets
-0x2900   message_class           octet    To send with a specific GatewayAPI messageclass
-======== ======================= ======== ========================
+======== ======================= ============= ========================
+Tag      Name                    Size          Description
+======== ======================= ============= ========================
+0x0005   dest_addr_subunit       1 byte        For "flash" sms
+0x0204   user_message_reference  2 bytes       User assigned reference for delivery reports
+0x0424   message_payload         1 byte        For sending messages longer than 255 octets
+0x2900   message_class           octetstring   To send with a specific GatewayAPI messageclass
```

Comment:
Do we have a length at all?

Reply:
I dont think we a length here. Just variable string.